### PR TITLE
Adds Append helper

### PR DESCRIPTION
**Pull request by maabernethy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -331,6 +332,16 @@ Checks if a given value or sub-array is contained within an array.
 
 **[⬆️ back to top](#available-helpers)**
 
+#### `append`
+Combines the given arrays and/or values into a single flat array.
```
Comment (poteto) [inline README.md]: Maybe `s/Combines/Appends`?

Comment (maabernethy) [inline README.md]: sure thang

**Comment by poteto** [comment]
Thanks for this PR!

**Comment by poteto** [comment]
@homu r+

**Comment by maabernethy** [comment]
@poteto No no... thank YOU!

**Pull request merged**